pandas/_libs/parsers.pyx

```diff
@@ -273,12 +275,16 @@ cdef extern from "parser/io.h":
 
 DEFAULT_CHUNKSIZE = 256 * 1024
 
+
+def c_type_conv(st):
+    cdef bytes py_bytes = st.encode()
```

Comment:
thanks, i think have found the function
_NA_VALUES = = _ensure_encoded(in parsers._NA_VALUES)
then even list comprehension is not needed; need to double check as haven't used Cython before